lib/constants.ts

```diff
@@ -28,6 +28,7 @@ export const CURSOR_COLOR = "hsl(47 100% 72%)";
 
 export const TRANSACTIONS_PAGE_SIZE = 20;
 
+export const TOP_CURRENCIES = ["USD", "EUR", "JPY", "CNY", "GBP"];
```

Comment:
This should come from the API shouldn't it? they have a higher preference (check the api response)

Reply:
there's a priority field in the response

Reply:
Ah thanks changed! Can you have a look again?